commit 7310b3c8e2df3ad9cac6cb540a3fb54d1e260123 (refs/changes/90/3190/1)
Author: Julien Gilli <julien.gilli@joyent.com>
Date:   2018-01-10T15:18:53-08:00 (1 year, 9 months ago)
    
    TRITON-45 VOLAPI uses wrong error level when logging successful import of storage VM image

diff --git a/lib/volapi.js b/lib/volapi.js
index e67abb9..f69dd1a 100644
--- a/lib/volapi.js
+++ b/lib/volapi.js
@@ -207,7 +207,7 @@ function importImage(imgapiClient, imageUuid, log, callback) {
                                 'Error when importing image');
                             importImgBackoff.backoff();
                         } else {
-                            log.error({imageUuid: imageUuid},
+                            log.info({imageUuid: imageUuid},
                                 'Imported image succesfully');
                             importImgBackoff.reset();
                             callback();
